### ISSUE TITLE
reloadCommand outside RainLoop code

### ISSUE THREAD

**Issue by Med-All** [RainLoop/rainloop-webmail#1221]
I'm trying to create a plugin for auto-refresh INBOX folder and any custom/shared folder after X time.

So far I have got it to work when user has selected INBOX or any custom/shared folder but when is for an example Trash is not taking any affect rather than is executed after random amount of time.

Is there anything in the RainLoop code what prevents to execute reloadCommand outside of main code for a period of time?

Thanks

**Comment by ecrazor1911** [comment]
+1

**Comment by ecrazor1911** [comment]
Workaround:

`$(".buttonReload").click();`

**Comment by Med-All** [comment]
That workaround is not perfect as this will keep tooltip "Reload" appear each time even when you are typing message :)

**Comment by ecrazor1911** [comment]
Yeah I know :( I don't like hacking solutions, but little CSS "hacking"?

`#opentip-5 { display: none!important; }`
